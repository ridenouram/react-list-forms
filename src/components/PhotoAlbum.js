import React from 'react';
import PropTypes from 'prop-types';
import Photos from './Photos';

export default function PhotoAlbum({ title, photos }) {
  return (
    <container>
      <h2>{title}</h2>
      <Photos photos={photos} />
    </container>
  );
}

PhotoAlbum.propTypes = {
  title: PropTypes.string.isRequired,
  photos: PropTypes.arrayOf(
    PropTypes.string.isRequired
  ).isRequired
};
