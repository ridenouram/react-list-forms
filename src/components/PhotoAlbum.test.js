import React from 'react';
import { shallow } from 'enzyme';
import PhotoAlbum from './PhotoAlbum';

describe('photo album component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<PhotoAlbum title='' photos={[]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
