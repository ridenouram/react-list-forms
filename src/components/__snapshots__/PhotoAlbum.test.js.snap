// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`photo album component matches snapshot 1`] = `
<container
  className="PhotoAlbum"
>
  <h2 />
  <Photos
    photos={Array []}
  />
</container>
`;
